tests/python/frontend/to_onnx/test_to_onnx.py

```diff
@@ -0,0 +1,262 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+"""Relay to ONNX serialization test cases"""
+import numpy as np
+import tvm
+from tvm import relay
+from tvm.relay import to_onnx
+import onnxruntime as rt
+
+
+def func_to_onnx(func, name):
+    mod = relay.Module()
+    mod['main'] = func
+    onnx_model = to_onnx.to_onnx(mod, {}, name, None)
+    return onnx_model.SerializeToString()
+
+
+def do_onnx_inference(onnx_model, input_data):
```

Comment:
how about `run_onnx`?

Reply:
me might need to add support for GPU

Reply:
@zhiics pls suggest the device list here we should support

Reply:
we can probably add whatever is available:

https://github.com/apache/incubator-tvm/blob/019356f810f7de27ba5f9f346f051d5b89a88c3e/python/tvm/relay/testing/config.py#L23